src/main/java/com/faforever/client/chat/AbstractChatTabController.java

```diff
@@ -212,6 +215,7 @@ public AbstractChatTabController(ClanService clanService, WebViewConfigurer webV
     String urlFormat = clientProperties.getVault().getReplayDownloadUrlFormat();
     String[] splittedFormat = urlFormat.split("%s");
     replayUrlPattern = Pattern.compile(Pattern.quote(splittedFormat[0]) + "(\\d+)" + Pattern.compile(splittedFormat.length == 2 ? splittedFormat[1] : ""));
+    channelUrlPattern = Pattern.compile("\\QfafJoinChannel://\\E(#[a-z].+)", CASE_INSENSITIVE);
```

Comment:
?

Reply:
they are, this is the constructor.

Reply:
axel1200 is right, this should be a constant like:
```
private static final Pattern CHANNEL_URL_PATTERN = Pattern.compile("\\QfafJoinChannel://\\E(#[a-z].+)", CASE_INSENSITIVE);
```

Otherwise, one pattern will be instantiated per instance of `AbstractChatTabController` even though they are identical. So we can just have a single, static instance.